apps/mobile/src/state/global-action.ts

```diff
@@ -2,7 +2,7 @@ import { createAction } from '@reduxjs/toolkit';
 
 import { BitcoinKeychainStore } from './keychains/bitcoin/bitcoin-keychains.slice';
 import { StacksKeychainStore } from './keychains/stacks/stacks-keychains.slice';
-import { WalletStore } from './wallets/wallets.slice';
+import { LooseWalletStore } from './wallets/wallets.slice';
```

Comment:
What does `Loose` mean here? Just double checking for my own learning

Reply:
Haha ok thanks! I was just reading it as `Lose` and then thought `Loose` could be some computer science term I didn't know 😄